cargo-doc-ngrok/Cargo.toml

```diff
@@ -8,6 +8,8 @@ repository = "https://github.com/ngrok/ngrok-rs"
 
 [dependencies]
 axum = "0.6.1"
+# use older bstr until watchexec::ignore-files switches from git-config 0.14.0 to gix-config 0.16.2 (PR 502)
+bstr = "=1.0.1"
```

Comment:
Is this going to throw udeps errors at us? If it does, might need to add it as a `[patch]` in the top level `Cargo.toml`.

Reply:
Udeps didn't error locally or in the [PR action](https://github.com/ngrok/ngrok-rs/actions/runs/4276444729/jobs/7444538360) which is grabbing this version. Could the check after merge behave differently?